File: projects/m1/003-making-change/solutions/dariogambarro/003-making-change.js
const penny = 0.01*100;
const nickel = 0.05*100;
const dime = 0.10*100;
const quarter = 0.25*100;
const loonie = 1*100;
const toonie = 2*100;
let change;
let numCoins = 0;

change = prompt("How much change I have to provide to you in cents?");

if (change % toonie != change) {                            // il resto deve essere diverso dal numeratore = controllo num > den
    numCoins = ((change - change % toonie) / toonie);       // sottraggo il resto della mod e divido per il valore della moneta
    console.log(numCoins, "toonie/s");                      // stampo il risultato
    change = (change%toonie);                               // nuovo valore del resto
}
if (change % loonie != change) {
    numCoins = ((change - change % loonie) / loonie);
    console.log(numCoins, "loonie/s");
    change = (change%loonie);
}
if (change % quarter != change) {
    numCoins = ((change - change % quarter) / quarter);
    console.log(numCoins, "quarter/s");
    change = (change%quarter);
}
if (change % dime != change) {
    numCoins = ((change - change % dime) / dime);
    console.log(numCoins, "dime/s");
    change = (change%dime);
}
if (change % nickel != change) {
    numCoins = ((change - change % nickel) / nickel);
    console.log(numCoins, "nickel/s");
    change = (change%nickel);
}
if (change % penny != change) {
    numCoins = ((change - change % penny) / penny);
    console.log(numCoins, "penny/s");
}

File: projects/m1/003-making-change/solutions/StefanoSantomaggio/003_making_change.py
cents_user = int(input('How many cents do you have? '))

toonie = cents_user / 200
cents_user %= 200

loonie = cents_user / 100
cents_user %=100

quarter = cents_user / 25
cents_user %= 25

dime = cents_user / 10
cents_user %= 10

nickel = cents_user / 5
cents_user %= 5

penny = cents_user / 1

print(f'Change: {int(toonie)} toonies, {int(loonie)} loonies, {int(quarter)} quarters, {int(dime)} dimes, {int(nickel)} nickels, {int(penny)} penny.')